crates/bevy_pbr/src/render/mesh_view_bindings.wgsl

```diff
@@ -46,37 +46,45 @@
 #endif
 @group(0) @binding(15) var environment_map_sampler: sampler;
 
-@group(0) @binding(16) var dt_lut_texture: texture_3d<f32>;
-@group(0) @binding(17) var dt_lut_sampler: sampler;
+#ifdef MULTIPLE_LIGHT_PROBES_IN_ARRAY
+@group(0) @binding(16) var irradiance_volumes: binding_array<texture_3d<f32>, 8u>;
```

Comment:
I'm worried we're going to go over the texture bindings on minspec devices. For transmission it was already causing issues. With irradiance volumes, reflection probes, and lightmaps, we're probably blowing past the limits...

Reply:
I was concerned about this too, but it did work on the Android simulator and WebGL 2. Is there some documentation on how to test on a min-spec device?

Reply:
I think you can configure the wgpu Limits as part of the WgpuSettings when configuring the plugins. Otherwise you can also just modify the initialize_render() function from bevy_render to use the minimum/default wgpu limits.

Reply:
What is the minimum limit that Bevy supports?